app/frontend/users/UserEdit.res

```diff
@@ -120,6 +144,29 @@ let uploadAvatar = (send, formData) => {
     () => send(SetAvatarUploadError(Some(t("upload_failed")))),
   )
 }
+
+let updateEmail = (send, email, newEmail) => {
+  SendEmailUpdateTokenQuery.fetch({newEmail: newEmail})
+  |> Js.Promise.then_((response: SendEmailUpdateTokenQuery.t) => {
+    response.sendUpdateEmailToken.success
+      ? {
+          Notification.success(
```

Comment:
It feels a bit odd to see that server generates the error notifications while success is by the client. Do move this notification into the mutation.